test/extended/two_node/tnf_recovery.go

```diff
@@ -234,6 +234,33 @@ var _ = g.Describe("[sig-etcd][apigroup:config.openshift.io][OCPFeatureGate:Dual
 			&nodeB, true, false, // member on node B expected started == true, learner == false
 			membersHealthyAfterDoubleReboot, pollInterval)
 	})
+
+	g.It("should recover from etcd process crash [Skipped:KnownIssue]", func() {
```

Comment:
This is not your doing, but I think we should add the [Serial] annotation to these recovery tests (in the root name) on line 45

Reply:
Fascinating. I wonder if they're added there because the suite is scheduled to run in serial?